### PR TITLE
FEAT: Support qwen2-vl-instruct GPTQ format and AWQ format

### PR DESCRIPTION
**Pull request by Jun-Howie**
No additional extensions are needed to support GPTQ, and AWQ formats, I have verified this locally!

### CODE REVIEW

File: xinference/model/llm/llm_family_modelscope.json [hunk 1]
```diff
@@ -4510,47 +4510,106 @@
...
-    "prompt_style": {
-      "style_name": "QWEN",
-      "system_prompt": "You are a helpful assistant",
-      "roles": [
-        "user",
-        "assistant"
-      ]
-    }
-  }
+	"version": 1,
```
[the remark this answers is not in the document]
Comment (Jun-Howie) [reply xinference/model/llm/llm_family_modelscope.json]: Completion

File: xinference/model/llm/llm_family.json [hunk 2]
```diff
@@ -6828,6 +6828,33 @@
...
+          "Int4"
+        ],
+        "model_id":"Qwen/Qwen2-VL-2B-Instruct-GPTQ-Int4",
+        "model_revision":"800d396518c82960ce6d231adecd07bbc474f0a9"
+      },
+         {
+        "model_format":"awq",
+        "model_size_in_billions":2,
+        "quantizations":[
+          "4-bit"
```
Comment (qinxuye) [inline xinference/model/llm/llm_family.json:6854]:
```suggestion
          "Int4"
```

File: xinference/model/llm/llm_family.json [hunk 3]
```diff
@@ -6837,6 +6864,33 @@
...
+          "Int4"
+        ],
+        "model_id":"Qwen/Qwen2-VL-7B-Instruct-GPTQ-Int4",
+        "model_revision":"5ab897112fa83b9699826be8753ef9184585c77d"
+      },
+        {
+        "model_format":"awq",
+        "model_size_in_billions":7,
+        "quantizations":[
+          "4-bit"
```
Comment (qinxuye) [inline xinference/model/llm/llm_family.json:6890]:
```suggestion
          "Int4"
```

File: xinference/model/llm/llm_family_modelscope.json [hunk 4]
```diff
@@ -4522,26 +4522,86 @@
...
+        ],
+        "model_hub": "modelscope",
+        "model_id":"qwen/Qwen2-VL-7B-Instruct-GPTQ-Int4",
+        "model_revision":"master"
+      },
+      {
+        "model_format":"awq",
+        "model_size_in_billions":7,
+        "quantizations":[
+          "4-bit"
```
Comment (qinxuye) [inline xinference/model/llm/llm_family_modelscope.json:4560]:
```suggestion
          "Int4"
```

File: xinference/model/llm/llm_family_modelscope.json [hunk 5]
```diff
@@ -4522,26 +4522,86 @@
...
+        ],
+        "model_hub": "modelscope",
+        "model_id":"qwen/Qwen2-VL-2B-Instruct-GPTQ-Int4",
+        "model_revision":"master"
+      },
+      {
+        "model_format":"awq",
+        "model_size_in_billions":2,
+        "quantizations":[
+          "4-bit"
```
Comment (qinxuye) [inline xinference/model/llm/llm_family_modelscope.json:4600]:
```suggestion
          "Int4"
```

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**